reference/orders.v2.oas2.yml

```
@@ -3979,7 +3535,7 @@ components:
           type: string
         customer_locale:
           type: string
-          example: \"en\", \"en-us\",
+          example: '\"en\", \"en-us\",'
```

I think this should just be a single string without the escape characters. The original swagger file was attempting to show multiple examples, but we should stick with a single example for the spec file. If I try to update an order with anything other than a single string such as `"en"`, `"en-au"`, or `"en-gb"`,  I get back the following error:

```json
  {
    "message": "The field 'customer_locale' is invalid.",
    "status": 400
  }
```

```suggestion
          example: 'en'
```